app/src/main/java/com/sunlightlabs/android/congress/BillPager.java

```diff
@@ -100,10 +100,7 @@ public void onLoadBill(Bill bill) {
 	}
 	
 	public void onLoadBill(CongressException exception) {
-		if (exception instanceof BillService.BillNotFoundException)
-			Utils.showEmpty(this, R.string.bill_not_known_yet);
-		else
-			Utils.showRefresh(this, R.string.bill_loading_error);
+		Utils.showRefresh(this, R.string.bill_loading_error);
```

Comment:
I'm comfortable ditching the specific exception class here, which I think was used to describe Sunlight-specific API behavior.